chains/moonbase/addresses.yaml

```diff
@@ -0,0 +1,13 @@
+domainRoutingIsmFactory: "0xA318540392ec7d0809B73B6D2065A50D7C62675B"
```

Comment:
could you also add the `merkleTreeHook` address please, can find by looking at deployment logs

and if you can't find an `interchainGasPaymaster`, set it to the zero address `0x0000000000000000000000000000000000000000`